rclcpp/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ cmake_minimum_required(VERSION 3.12)
 
 project(rclcpp)
 
+option(DEFINE_CONTENT_FILTER "Content filter feature support" ON)
+if(DEFINE_CONTENT_FILTER)
+  add_definitions(-DCONTENT_FILTER_ENABLE)
+endif()
+
```

Comment:
So this will be always on as ROS 2 rclcpp library, but user explicitly enable this when they want to use ContentFitleredTopic, otherwise build fails. my understanding correct? @wjwwood could we have feedback on this, if this is what we want?

Reply:
As @ivanpauno mentioned, this isn't really accomplishing what we wanted, so let's just remove it and provide access to CFT uniformly for now. We'll just have to document the known issues.